src/aiida/schedulers/plugins/slurm.py

```diff
@@ -207,7 +207,7 @@ def _get_joblist_command(self, jobs=None, user=None):
             else:
                 if not isinstance(jobs, (tuple, list)):
                     raise TypeError("If provided, the 'jobs' variable must be a string or a list of strings")
-                joblist = jobs
+                joblist = list(jobs)
```

Comment:
Maybe just add a comment here why this is needed (to make sure nobody removes this :-) )